client/src/components/login/Login.js

```diff
@@ -1,50 +1,62 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
-import {Form,Label,Input,Button} from './../../theme/theme.js';
-
-
-const H2=styled.h2`
-    text-align:center;`
-
-    const P=styled.p`
-    margin-left:25%;
-    font-size:12px;   
-    &:hover{
-        text-decoration-color:gray;
-        text-decoration:underline;
-        cursor:pointer;
-    }
-    `
-
-    const FormWrap=styled.div`
-    margin-top:30px;
-    margin-left:25%;
-    `
+import { Form, Label, Input, Button } from './../../theme/theme.js';
+
+const H2 = styled.h2`
+  text-align: center;
+  font-size: 2.2rem;
+`;
+
+const P = styled.p`
+  margin-left: 25%;
+  font-size: 12px;
+  &:hover {
+    text-decoration-color: gray;
+    text-decoration: underline;
+    cursor: pointer;
+  }
+`;
+
+const FormWrap = styled.div`
+  margin-top: 30px;
+  margin-left: 25%;
+`;
 
 function Login(props) {
-    const [name, setName] = useState('');
-    const [password, setPassword] = useState('');
-    const [email,setEmail]=useState('');
-
-    return (
-
-            <Form background="white" >
-                <H2 >Login </H2>
-                <FormWrap>
-                <Label htmlFor="email">Email:<br />
-                <Input type="text" name="email" id="email" required /><br />
-                </Label>
-                <Label htmlFor="password">Password:<br />
-                <Input type="text" name="password" id="password" required /><br />
-                </Label>
-                <Button background="purple" type="submit">Sign In</Button>
-                </FormWrap>
-                <hr />
-                <P onClick={()=>props.set_new_user(true)}>Register</P>
-            </Form>
-
-    )
+  const [name, setName] = useState('');
+  const [password, setPassword] = useState('');
+  const [email, setEmail] = useState('');
+
+  return (
+    <Form background="white">
+      <H2>Login </H2>
+      <FormWrap>
+        <Label htmlFor="email">
+          Email:
+          <br />
+          <Input
+            type="email"
+            name="email"
```

Comment:
You don't need to use `htmlFor="email"` in this setup, you also delete `name="email"` in the input.  Also apply these changes to the other label/input areas.

https://reactjs.org/docs/forms.html

Reply:
The reason I have htmlFor is that some people like to click on the label. Once people click on the label the cursor will go to the respective input field and it is pretty standard when you build a form. However I can take the "name" off .

Reply:
`htmlfor` is needed for accessibility as well

Reply:
@a-w-m @Junjpan ah gotcha